devices/common_patches/h7_exti_dualcore.yaml

```diff
@@ -0,0 +1,853 @@
+# H7 EXTI module, for dual core parts
+
+EXTI:
+  _modify:
+    C1PR2,C1IMR3,C1EMR3,C1PR3:
+      access: read-write
+
+  _add:
+    C2IMR1:
+      description: EXTI interrupt mask register
+      addressOffset: 0x00C0
+      resetValue: 0xFFC00000
+      fields:
+        MR0:
+          description: CPU2 rising trigger event configuration bit of Configurable Event input
```

Comment:
I think all these descriptions are wrong, RM says "CPU2 interrupt mask on [direct|configurable] event input x" (direct for 22-31, configurable for 0-21)

Reply:
Fixed